bundles/org.eclipse.equinox.simpleconfigurator/src/org/eclipse/equinox/internal/simpleconfigurator/ConfigApplier.java

```diff
@@ -115,11 +120,91 @@ void install(URL url, boolean exclusiveMode) throws IOException {
 					if (additionalRefresh.length > 0)
 						refreshPackages(additionalRefresh, manipulatingContext);
 				}
+			}
+			// when refreshing large sets of bundles the resolver sometimes take a choice
```

Comment:
Do you have a scenario that exhibits this behavior?  Is there going to be a way to add a test for this?

Reply:
Yes I have a product where I install a larger set of plugins into an older Eclipse version, but I'm not sure how this cane be transformed into a standalone test-case, do we have such facility somewhere?

I could think about exporting all bundles into a OSGi Repository and then call the resolver api with this repository as a source...